stubs/fpdf2/fpdf/linearization.pyi

```diff
@@ -34,7 +34,7 @@ class PDFXrefAndTrailer(ContentWithoutID):
     def is_first_xref(self) -> bool: ...
     @property
     def is_main_xref(self) -> bool: ...
-    def serialize(self) -> str: ...
+    def serialize(self, _security_handler: object = None) -> str: ...
```

Comment:
Should this be `StandardSecurityHandler | None` as well, like all the other `_security_handler` parameters?

Reply:
Ah yes, I knew that I missed one.